itou/users/management/commands/send_users_to_brevo.py

```diff
@@ -154,9 +175,54 @@ def import_prescribers(self, client, professional_qs, *, wet_run):
         prescribers = list(all_prescribers.filter(Exists(authorized_prescriber_memberships)))
         logger.info("Prescribers count: %d", len(prescribers))
         if wet_run:
-            client.import_users(prescribers, authorized_prescriber_serializer)
+            client.import_users(prescribers, BREVO_LES_EMPLOIS_LIST_ID, authorized_prescriber_serializer)
 
         orienteurs = list(all_prescribers.exclude(Exists(authorized_prescriber_memberships)))
         logger.info("Orienteurs count: %d", len(orienteurs))
         if wet_run:
-            client.import_users(orienteurs, prescriber_serializer)
+            client.import_users(orienteurs, BREVO_LES_EMPLOIS_LIST_ID, prescriber_serializer)
+
+    def import_job_seekers(self, client, *, wet_run):
+        job_seekers = User.objects.filter(
+            Exists(
+                EmailAddress.objects.filter(
+                    user_id=OuterRef("pk"),
+                    email=OuterRef("email"),
+                    primary=True,
+                    verified=True,
+                )
+            ),
+            kind=UserKind.JOB_SEEKER,
+            is_active=True,
+        )
+
+        midnight_today = datetime.datetime.combine(
+            timezone.localdate(),
+            datetime.time.min,
+            tzinfo=timezone.get_current_timezone(),
+        )
+        a_month_ago = midnight_today - datetime.timedelta(days=30)
+        recently_joined = job_seekers.filter(date_joined__gte=a_month_ago)
+        logger.info("Job seekers count: %d", len(recently_joined))
+        if wet_run:
+            client.import_users(recently_joined, BREVO_CANDIDATS_LIST_ID, job_seeker_serializer)
+
+        six_months_ago = midnight_today - datetime.timedelta(days=182)
+        job_apps_subquery = JobApplication.objects.filter(job_seeker=OuterRef("pk"))
+        stalled_autonomous_job_seekers = (
+            job_seekers.filter(
+                Exists(job_apps_subquery.filter(sender=OuterRef("pk"), created_at__lte=a_month_ago)),
+                Exists(job_apps_subquery.filter(sender=OuterRef("pk"), created_at__gte=six_months_ago)),
+            )
```

Comment:
```suggestion
            job_seekers.filter(
                Exists(job_apps_subquery.filter(
                    sender=OuterRef("pk"),
                    created_at__lte=a_month_ago,
                    created_at__gte=six_months_ago
                )
            )
```
C'est peu être lié à la discussion en apérothèque d'hier, mais pourquoi ne pas faire ça ?
J'ai l'impression que tu veux un candidat avec au moins une candidature entre il y a 6 mois et il y a un mois, mais j'ai l'impression que tu vas matcher un candidat avec une candidature il y a 7 mois et une hier.

Reply:
C'est quoi la définition d'un candidat bloqué ? Dans la cart notion je ne vois que la définition de candidat dans la file active

Reply:
Oui, avec la règle une de plus de 6 mois et une de moins d'un mois, ma suggestion ne marche plus, c'est pour ça que j'ai eu un doute et que j'ai voulu vérifier la définition ^^

Reply:
Non non, j’avais bien fait ce qu’il faut. Crée avant `a_month_ago`, donc il y a plus de 1 mois, et après `six_month_ago`, donc il y a moins de 6 mois.

Reply:
J’ai aussi ajouté un cas de test pour une seule candidature correspondant et au critère <1m et au critère >6m.

Reply:
Avec ce queryset, tu vas matcher à la fois : 
- Un candidat avec seulement une candidature il y a 1 mois
- Un candidat avec seulement une candidature il y a 6 mois
- Un candidat avec une candidature il y a 7 mois et une autre hier.

Pour être dans la file active, il faut avoir une candidature dans les 6 derniers mois, mais pas dans le dernier mois

```
            job_seekers.filter(
                Exists(job_apps_subquery.filter(sender=OuterRef("pk"), created_at__gte=six_months_ago)),
            ).exclude(
                Exists(job_apps_subquery.filter(sender=OuterRef("pk"), created_at__gte=a_month_ago)),
            )
```

Reply:
En effet, j'avais mal compris la règle :+1: